manpages/burp.8.in

```diff
@@ -138,6 +138,9 @@ Log file for status monitor - useful for debugging.
 \fB@name@ \-a b\fR
 Runs a backup.
 .TP
+\fB@name@ \-a t\fR
```

Comment:
I don't think it is correct to copy and paste the full description of '-a t' into the example section. If it is in the examples section at all, it should just say 'Runs a timed backup.'

Reply:
Ah sorry, I see that you have moved the whole section here to try to make it flow better.